modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/components/OrderableTable.tsx

```diff
@@ -217,22 +216,74 @@ const OrderableTableRow = ({
 	);
 };
 
+const Table = ({
+	actions,
+	fields,
+	items,
+	onDragCrossover,
+	onDrop,
+	query,
+}: {
+	actions?: Array<IAction>;
+	fields: Array<IField>;
+	items: Array<any>;
+	onDragCrossover: Function;
+	onDrop: Function;
+	query: string;
+}) => {
+	const [, dropRef] = useDrop({
+		accept: ROW_DRAGGABLE,
+		drop() {
+			onDrop();
+		},
+	});
+
+	return (
+		<ClayTable className="orderable-table" ref={dropRef}>
+			<ClayTable.Head>
+				<ClayTable.Row>
+					<ClayTable.Cell className="drag-handle-cell" />
+
+					{fields.map((field) => (
+						<ClayTable.Cell headingCell key={field.name}>
+							{field.label}
+						</ClayTable.Cell>
+					))}
+
+					{actions && <ClayTable.Cell className="actions-cell" />}
+				</ClayTable.Row>
+			</ClayTable.Head>
+
+			<ClayTable.Body>
+				{items.map((item, index) => (
+					<Row
+						actions={actions}
+						fields={fields}
+						index={index}
+						item={item}
+						key={item.id || index}
+						onDragCrossover={onDragCrossover}
+						query={query}
+					/>
+				))}
+			</ClayTable.Body>
+		</ClayTable>
+	);
+};
+
 interface IOrderableTableProps {
 	actions?: Array<IAction>;
 	className?: string;
 	creationMenuItems?: React.ComponentProps<
 		typeof ClayDropDownWithItems
 	>['items'];
 	creationMenuLabel?: string;
-	disableSave?: boolean;
 	fields: Array<IField>;
 	items: Array<any>;
 	noItemsButtonLabel: string;
 	noItemsDescription: string;
 	noItemsTitle: string;
-	onCancelButtonClick: Function;
-	onOrderChange: (args: {orderedItems: any[]}) => void;
-	onSaveButtonClick: Function;
+	onOrderChange: (args: {order: string}) => void;
```

Comment:
`onOrderChange` stays as a contract, triggered on drop. I did simplify it to return a CSV string, so we don't have to do it externally in each use. I think this is fine, because we are always using the same persistence. If `OrderableTable` was more general-purpose component, used outside of DSM, this would not be ideal.